src/main/webapp/components/workspace/workspace.js

```diff
@@ -160,9 +172,50 @@ export default () => {
             overflow: 'auto',
           }}
         >
-          <Typography variant="h4" component="h1" style={{ padding: 20 }}>
-            {title}
-          </Typography>
+          <div
+            style={{
+              display: 'flex',
+              alignItems: 'baseline',
+            }}
+          >
+            <TextField
+              variant="outlined"
+              style={{ margin: '10px', width: '70%' }}
+              value={title}
+              onChange={e => setTitle(e.target.value)}
```

Comment:
I don't think we should introduce state for title. I would suggest using an uncontrolled input here by using a `defaultValue` prop of `attributes.title` and having the onBlur reference the target value. i.e. 
```
onBlur={e=>saveWorkspace(..., {title: e.target.value} ...)}
```

Reply:
done 👍